webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.java

```diff
@@ -409,33 +409,32 @@ public void renderHead(IHeaderResponse aResponse)
         aResponse.render(JavaScriptHeaderItem.forReference(BratVisualizerUiResourceReference.get()));
         aResponse.render(JavaScriptHeaderItem.forReference(BratAnnotatorUiResourceReference.get()));
         //aResponse.render(JavaScriptHeaderItem.forReference(BratUrlMonitorResourceReference.get()));
-        
-        StringBuilder script = new StringBuilder();
+
```

Comment:
Can you please undo the changes to the string building here? The comments are no longer where they belong.